iosApp/iosApp/Icons.xcassets/accessibility-icon-alert.imageset/Contents.json

```diff
@@ -11,12 +11,15 @@
           "value": "dark"
         }
       ],
-      "filename": "elevator-alert-dark.svg",
+      "filename": "accessibility-icon-alert-dark.svg",
       "idiom": "universal"
     }
   ],
   "info": {
     "author": "xcode",
     "version": 1
+  },
+  "properties": {
+    "preserves-vector-representation": true
```

Comment:
TIL: this checkbox isn't ticked by default, but makes SVGs scale nicely at large sizes

Reply:
Oh, in that case we could set this on `accessibility-icon-accessible` and delete `accessibility-icon-accessible-large`.